config/config.default.json

```diff
@@ -17,7 +17,9 @@
     },
     "UsersConfiguration": {
         "InitialActiveUsers": 4,
-        "MaxActiveUsers": 1000
+        "MaxActiveUsers": 1000,
+        "UseMultipleSessionsPerUser": false,
+        "AvgSessionsPerUser": 4,
```

Comment:
I thought that if we made `AvgSessionsPerUser` default to 1 we could remove `UseMultipleSessionsPerUser`.

Reply:
Ah yes. Didn't realize you wanted that implicit.